### PR TITLE
feat: Only display CTA application on school page if displayOnWebsite…

### PR DESCRIPTION
**Pull request by annalinneajohansson**
… is true

### CODE REVIEW

File: library/Controller/ContentType/School/SchoolDataPreparer.php [hunk 1]
```diff
@@ -278,7 +278,7 @@ private function appendViewApplicationData(): void
     {
         $this->data['application'] = [];
 
-        $this->data['application']['displayOnWebsite'] = (bool) $this->postMeta->ctaApplication->display_on_website;
+        $this->data['application']['displayOnWebsite'] = (bool) $this->postMeta->ctaApplication->display_on_website ?: true;
```
Comment (thorbrink) [inline library/Controller/ContentType/School/SchoolDataPreparer.php:281]: Would not this condition always make `$this->data['application']['displayOnWebsite']` true?

**Review by thorbrink — CHANGES_REQUESTED** [review]

File: library/Controller/ContentType/School/SchoolDataPreparer.php (hunk 1 above)
Comment (annalinneajohansson) [reply library/Controller/ContentType/School/SchoolDataPreparer.php:281]: Ah, you're right! I only want to check it it's an empty string. I'll fix that.

Comment (thorbrink) [reply library/Controller/ContentType/School/SchoolDataPreparer.php:281]: Can it not also be `null`?

**Review by thorbrink — APPROVED** [review]
👍

**Pull request merged**